.github/workflows/build.yml

```diff
@@ -159,10 +159,12 @@ jobs:
     - uses: actions/checkout@v3
     - name: Prepare homebrew environment variables
       run: |
-        echo "LDFLAGS=-L$(brew --prefix tcl-tk)/lib" >> $GITHUB_ENV
-        echo "PKG_CONFIG_PATH=$(brew --prefix openssl@1.1)/lib/pkgconfig:$(brew --prefix tcl-tk)/lib/pkgconfig" >> $GITHUB_ENV
+        brew install pkg-config openssl@1.1 xz gdbm tcl-tk
+        echo "CFLAGS=-I$(brew --prefix gdbm)/include -I$(brew --prefix xz)/include" >> $GITHUB_ENV
+        echo "LDFLAGS=-L$(brew --prefix gdbm)/lib -I$(brew --prefix xz)/lib" >>  $GITHUB_ENV
+        echo "PKG_CONFIG_PATH=$(brew --prefix tcl-tk)/lib/pkgconfig" >> $GITHUB_ENV
```

Comment:
Why not follow the devguide recipe and use one shell line? Those variables should only be needed for the configure command

Reply:
@ned-deily 
* multi line string needs to declare sperate variable for this: https://trstringer.com/github-actions-multiline-strings/
So we use single line command most of cases.

Reply:
> * multi line string needs to declare sperate variable for this: https://trstringer.com/github-actions-multiline-strings/

But the devguide recipe does not consist of multiple commands on multiple lines as in that link, i.e. it's not "in the form of an output from one step, and an input to another step". It's a single shell command that has been split across lines in the devguide for readability and it doesn't need to be. (The POSIX definition calls it a "simple command": "A `simple command` is a sequence of optional variable assignments and redirections, in any sequence, optionally followed by words and redirections, terminated by a control operator." There is no piping involved here.)